LICENSE

```diff
@@ -1106,3 +1107,101 @@ Library.
     of your accepting any such warranty or additional liability.
 
 -----------------------------------------------------------------------------
+
```

Comment:
No need for this license if it comes through a gem, we only include licenses for stuff we directly vendor in the repo